packages/snfoundry/scripts-ts/helpers/fees.ts

```diff
@@ -83,6 +84,24 @@ export async function getBalance(
     return uint256.uint256ToBN(balance);
 }
 
-function getTxVersionFromFeeToken(feeToken: string) {
-    return feeToken === "strk" ? "0x3" : "0x1";
+function getTxVersionFromFeeToken(feeToken: string, isSierra?: boolean) {
+    return feeToken === "strk" ? ETransactionVersion.V3 :
+        isSierra ? ETransactionVersion.V2 :
+            ETransactionVersion.V1;
 }
+
+
+/**
+ * V_ Transaction versions HexString
+ * F_ Fee Transaction Versions HexString (2 ** 128 + TRANSACTION_VERSION)
+ */
+export enum ETransactionVersion {
```

Comment:
To standardize please remove any hungarian conventions like `I` or `E`, can write this as just `TransactionVersion`